helm-chart/renku/templates/ingress.yaml

```diff
@@ -59,53 +53,54 @@ spec:
             port:
               name: {{ $keycloakServicePort }}
       {{- end }}
-      - path: /gitlab
+      - path: /api
         pathType: Prefix
         backend:
           service:
             name: {{ template "renku.fullname" $ }}-gateway
             port:
               number: 80
-      - path: /repos
```

Comment:
Moved further down in the file in an if statement that templates this only when v1 services are enabled.